plugins/modules/net_tools/gandi_livedns.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019 Gregory Thiemonge <gregory.thiemonge@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: gandi_livedns
+author:
+- Gregory Thiemonge (@gthiemonge)
+requirements:
+- python >= 2.6
+version_added: "1.4.0"
+short_description: Manage Gandi LiveDNS records
+description:
+- "Manages dns records via the Gandi LiveDNS API, see the docs: U(https://doc.livedns.gandi.net/)"
+options:
+  api_key:
+    description:
+    - Account API token.
+    type: str
+    required: true
+  record:
+    description:
+    - Record to add.
+    - Default is C(@).
+    type: str
+    default: '@'
```

Comment:
Out of curiosity: what does `@` stand for? (And why is it the default?) Is it what other DNS modules would call `''`, i.e. edit the DNS entries for `domain` and not a subdomain?

Reply:
Yes, '@' represents the DNS entry for the domain in the Gandi API, I'll update the default to '', to be consistent with other DNS modules